docs/guide/website/sidebars.json

```diff
@@ -34,8 +34,18 @@
       "channels/smooch",
       "channels/faq"
     ],
-    "Tutorials": [
-      "tutorials/cluster-digital-ocean",
+    "Deployment And Infrastructure": [
+      "infrastructure/deploying",
+      "infrastructure/hosting",
+      "infrastructure/performances",
+      "infrastructure/pipeline",
+      "infrastructure/rbac",
```

Comment:
This should be placed within the bots and user management IMO. I know the GSheet says otherwise. @EFF What's your take on this?

Reply:
RBAC isn't infrastructure related indeed. We don't have a section for this but it would be "collaboration" so yes you can move this on bot building

Reply:
Also you have a pro/rbac section which is the same I guess ?

Reply:
@Michael-N-M Let's keep the RBAC where it is now, and when we are ready to make the Managing chatbots and users section, we'll transfer it there.